app/views/shared/provider/navigation/service/_integration.html.slim

```diff
@@ -6,6 +6,10 @@
          title: 'Methods & Metrics',
          path: admin_service_metrics_path(@service)
 
+= render vertical_nav_item,
```

Comment:
As we spoke before, let's not forget to hide from the menu if the rolling update is disabled.